OZprivate/ServerScripts/TaxonMappingAndPopularity/README.markdown

```diff
@@ -52,4 +52,4 @@ The script takes the following steps, as defined in the `main()` routine at the
 	* `calc_popularities_for_wikitaxa()` - use the page sizes and visit stats to calculate a raw popularity score for each taxon.
 	* `inherit_popularity()` - percolate popularity stats through the tree, to create "phylogeneticlaly informaed popularity".
 
-7. `output_simplified_tree()` - parse the tree and the taxa (e.g. removing monotomies, adding popularity rankings, etc), then save the results to a CSV file, and the final tree to a default location. The tree structure is also present in the CSV file in a ordered_set notation.
\ No newline at end of file
+7. `output_simplified_tree()` - parse the tree and the taxa (e.g. removing monotomies, adding popularity rankings, etc), then save the results to a CSV file, and the final tree to a default location. The tree structure is also present in the CSV file in a ordered_set notation.
```

Comment:
Not sure why it shows a diff here, I didn't change this line...

Reply:
Weird. Guess we can ignore that though. Thanks for the update @davidebbo